pkg/standalone/standalone.go

```diff
@@ -612,41 +612,45 @@ func untar(filepath, targetDir, binaryFilePrefix string) (string, error) {
 
 	tr := tar.NewReader(gzr)
 
+	foundBinary := ""
 	for {
 		header, err := tr.Next()
 
-		switch {
-		case err == io.EOF:
-			return "", fmt.Errorf("file is empty")
-		case err != nil:
+		if err == io.EOF {
+			break
+		} else if err != nil {
 			return "", err
-		case header == nil:
+		} else if header == nil {
 			continue
 		}
 
-		extractedFilePath := path.Join(targetDir, header.Name)
-
-		switch header.Typeflag {
-		case tar.TypeReg:
-			// Extract only the binaryFile
-			if header.Name != binaryFilePrefix {
-				continue
-			}
-
-			f, err := os.OpenFile(extractedFilePath, os.O_CREATE|os.O_RDWR, os.FileMode(header.Mode))
-			if err != nil {
+		// untar all files in archive
+		path := path_filepath.Join(targetDir, header.Name)
+		info := header.FileInfo()
+		if info.IsDir() {
+			if err = os.MkdirAll(path, info.Mode()); err != nil {
 				return "", err
 			}
+			continue
+		}
 
-			// #nosec G110
-			if _, err := io.Copy(f, tr); err != nil {
-				return "", err
-			}
-			f.Close()
+		f, err := os.OpenFile(path, os.O_CREATE|os.O_RDWR, os.FileMode(header.Mode))
+		if err != nil {
+			return "", err
+		}
+		defer f.Close()
+
+		// #nosec G110
+		if _, err = io.Copy(f, tr); err != nil {
+			return "", err
+		}
 
-			return extractedFilePath, nil
+		// If the found file is the binary that we want to find, save it and return later
+		if strings.HasSuffix(header.Name, binaryFilePrefix) || strings.HasSuffix(header.Name, binaryFilePrefix+".exe") {
```

Comment:
runtime.GOOS is windows is handled by unzip method

Reply:
The current artifacts published for Dashboard are tarfiles for both windows and linux, So I had to make adjustments to this

Reply:
The artifact for dashboard on windows is now a zip archive, so I updated this.